Content.Server/Medical/DefibrillatorSystem.cs

```diff
@@ -77,7 +77,7 @@ private void OnDoAfter(EntityUid uid, DefibrillatorComponent component, Defibril
         Zap(uid, target, args.User, component);
     }
 
-    public bool CanZap(EntityUid uid, EntityUid target, EntityUid? user = null, DefibrillatorComponent? component = null)
+    public bool CanZap(EntityUid uid, EntityUid target, EntityUid? user = null, DefibrillatorComponent? component = null, bool targetCanBeAlive = false)
     {
         if (!Resolve(uid, ref component))
```

Comment:
add a param comment to targetcanbealive me too tired to understand what it even means.